package/dust/sclang_conf.yaml

```diff
@@ -0,0 +1,5 @@
+includePaths:
```

Comment:
Still need to fix https://github.com/monome/norns/issues/799 to get rid of this file.

I think it should be possible using `recompile` from within sclang, but we'll need to figure out a way to add the path, recompile and only then run `Crone.initClass`.